### PR TITLE
update credits

### PR DESCRIPTION
**Pull request by cmkohnen**
Append "et al." to the author name

Also add Design section

### CODE REVIEW

**Comment by cmkohnen** [comment]
Don't know why CI fails but it all seems related to outside of our project

**Comment by Sjoerd1993** [comment]
Was just about to comment the same about the CI, these changes are good by me. Just pinging @bertob, so that he's aware that he's credited for his efforts on this front, and we don't put someone's name on this without their knowledge. 

Also, in case you'd prefer to not have your name on the credits section, just let use know @bertob

**Review by Sjoerd1993 — APPROVED** [review]

**Comment by Sjoerd1993** [comment]
Getting some weird Github errors anyway on the webpage, so that could be related to the CI failing.

File: src/ui.py [hunk 1]
```diff
@@ -172,11 +172,17 @@ def show_about_window(self):
...
+        developer_name=self.get_author() + ", et al.",
+        issue_url=self.get_issues(),
         version=self.get_version(), developers=[
             "Sjoerd Stendahl <contact@sjoerd.se>",
             "Christoph Kohnen <christoph.kohnen@disroot.org>",
         ],
+        designers=[
+            "Sjoerd Stendahl <contact@sjoerd.se>",
+            "Christoph Kohnen <christoph.kohnen@disroot.org>",
+            "Tobias Bernard",
```
Comment (Sjoerd1993) [inline src/ui.py:184]: Perhaps add email here as well for consistency's sake. (From [here](https://tobiasbernard.com/))

```suggestion
            "Tobias Bernard", <hi@tobiasbernard.com>",
```

Comment (bertob) [reply src/ui.py:184]: No objection to being mentioned, though I'd prefer to have my GNOME email address in there (tbernard@gnome.org) :)

**Pull request merged**